charts/lagoon-core/templates/insights-handler.deployment.yaml

```diff
@@ -49,14 +49,6 @@ spec:
           value: {{ required "A valid .Values.s3FilesBucket required!" .Values.s3FilesBucket | quote }}
         - name: S3_FILES_HOST
           value: {{ required "A valid .Values.s3FilesHost required!" .Values.s3FilesHost | quote }}
-        {{- with .Values.s3FilesAccessKeyID }}
-        - name: S3_FILES_ACCESS_KEY_ID
-          value: {{ . | quote }}
-        {{- end }}
-        {{- with .Values.s3FilesSecretAccessKey }}
-        - name: S3_FILES_SECRET_ACCESS_KEY
-          value: {{ . | quote }}
-        {{- end }}
```

Comment:
Could we specify the individual env vars here instead of importing all those in the api secret? Reasons are:
* makes it easy to see at a glance which variables the pod actually uses
* sensitive info that the pod doesn't need is kept out of the pod's environment

What I mean is to do something like this instead: https://github.com/uselagoon/lagoon-charts/blob/aa9ce769295844b07b081fefe602e3e45c355105/charts/lagoon-core/templates/api-redis.deployment.yaml#L33-L38

Reply:
Sure